File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverHere/LockupEntrypoint/Media.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Media {

    @SerializedName("images")
    @Expose
    private Images images;
    @SerializedName("reviews")
    @Expose
    private Reviews reviews;
    @SerializedName("ratings")
    @Expose
    private Ratings ratings;

    public Images getImages() {
        return images;
    }

    public void setImages(Images images) {
        this.images = images;
    }

    public Reviews getReviews() {
        return reviews;
    }

    public void setReviews(Reviews reviews) {
        this.reviews = reviews;
    }

    public Ratings getRatings() {
        return ratings;
    }

    public void setRatings(Ratings ratings) {
        this.ratings = ratings;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/LockupEntrypoint/Media.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint

import com.google.gson.annotations.Expose

class Media {
    @SerializedName("images")
    @Expose
    private var images: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Images? = null

    @SerializedName("reviews")
    @Expose
    private var reviews: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Reviews? = null

    @SerializedName("ratings")
    @Expose
    private var ratings: Ratings? = null
    fun getImages(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Images? {
        return images
    }

    fun setImages(images: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Images?) {
        this.images = images
    }

    fun getReviews(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Reviews? {
        return reviews
    }

    fun setReviews(reviews: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere.LockupEntrypoint.Reviews?) {
        this.reviews = reviews
    }

    fun getRatings(): Ratings? {
        return ratings
    }

    fun setRatings(ratings: Ratings?) {
        this.ratings = ratings
    }
}